transport/server.cc

```diff
@@ -207,11 +207,68 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
         sm::make_derive("requests_shed", _stats.requests_shed,
                         sm::description("Holds an incrementing counter with the requests that were shed due to overload (threshold configured via max_concurrent_requests_per_shard). "
                                             "The first derivative of this value shows how often we shed requests due to overload in the \"CQL transport\" component.")),
-       sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
+        sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
                         sm::description(
                             seastar::format("Holds the amount of available memory for admitting new requests (max is {}B)."
                                             "Zero value indicates that our bottleneck is memory and more specifically - the memory quota allocated for the \"CQL transport\" component.", _max_request_size))),
+        
+        sm::make_derive("server_errors", _stats.errors[exceptions::exception_code::SERVER_ERROR],
```

Comment:
I hope that someone who's an expert in the overall monitoring "experience" (@amnonh ?) can review the naming of these counters. Something that bothers me is that you're adding about 20 new metrics which have a common theme (types of CQL errors) and yet there is no common naming that could make that clear. Maybe we should name all of them "cql_error_*"? Or maybe they should all be one metric with different labels - something which will also allow the user to request the sum of all errors?
I'm just raising a question here. I don't know the answer.